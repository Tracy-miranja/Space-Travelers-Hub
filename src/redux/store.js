import { configureStore } from '@reduxjs/toolkit';
import missionReducer from './Mission/MissionsSlice';
import rocketReducer from './Rocket/RocketsSlice';

export default configureStore({
  reducer: {
    mission: missionReducer,
    rockets: rocketReducer,
  },
});
