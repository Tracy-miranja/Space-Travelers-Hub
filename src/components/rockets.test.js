import React from 'react';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import Rockets from './rockets';
import store from '../redux/store';

describe('Rockets', () => {
  it('should match the snapshot', () => {
    const { container } = render(
      <Provider store={store}>
        <Rockets />
      </Provider>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
