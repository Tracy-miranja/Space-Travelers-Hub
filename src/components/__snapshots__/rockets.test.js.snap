// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets should match the snapshot 1`] = `
<div>
  <img
    alt="loading..."
    class="loader"
    src="loader.gif"
  />
</div>
`;
